.mergify.yml

```diff
@@ -17,9 +17,15 @@ pull_request_rules:
       - label!=blocked
       - label!=no-mergify
       - head~=^(?!release.*).*$
+  - name: recreate dependabot PRs with merge conflicts
+    conditions:
+      - "status-success=ci/circleci: debug-build-test"
+      - "status-success=license/cla"
```

Comment:
Maybe remove this one as we don't care if licence service is down, we should still recreate.

Reply:
Fixed!